tooling/template-base/components/LearnerAccess/ListDisplayDropDown.tsx

```diff
@@ -0,0 +1,54 @@
+import {
+  HydratedContentItem,
+  useUserCourseCompletionProgressQuery
+} from '@thoughtindustries/content';
+import React from 'react';
+interface ContentUiProps {
+  item: HydratedContentItem;
+  index?: number;
+}
+
+const LearnerAccessListDisplayDropDown = ({ item }: ContentUiProps) => {
+  const { data } = useUserCourseCompletionProgressQuery({
+    variables: {
+      id: item.id
+    }
+  });
+  return (
+    <>
+      <hr className="bg-gray-500"></hr>
+      <div className="flex flex-col bg-slate-50 p-6 space-y-6 border-l-4 border-blue-700">
+        {/* article and duration */}
+        <div className="flex flex-row">
+          <div className="py-1 px-3 rounded-full bg-green-200">
+            <div className="text-sm font-semibold">{item.contentTypeLabel}</div>
+          </div>
+          <div className="flex items-center text-gray-500 px-4">|</div>
+          <div className="flex items-center text-gray-500 font-sm font-semibold">30 mins</div>
```

Comment:
Think you might mean`item.contentTime` or `item.contentEstimate` here?

Reply:
discussed this with Chris he said "there is no such property, custom fields could perhaps be used, if you add that custom field to your instance", that's why I just left the placeholder for the moment.

`item.contentTime` or `item.contentEstimate` don't seem to exist on my hydrated Item. Should I create a custom field in my instance or would it be best to omit this?